src/Avalonia.Controls/ContextMenu.cs

```diff
@@ -302,11 +337,7 @@ public void Open(Control? control)
                 ((ISetLogicalParent)_popup).SetParent(control);
             }
 
-            if (PlacementTarget is null && _popup.PlacementTarget != control)
-            {
-                _popup.PlacementTarget = control;
-            }
-
+            _popup.PlacementTarget = placementTarget;
```

Comment:
It seems that `ContextMenu.PlacementTarget` property is now unused. What if somebody is manually assigning placement target, wouldn't this ignore it?

Reply:
Good catch 👍